patches/nuget-client/0001-Pin-newtonsoft-json-to-a-specific-version.patch

```diff
@@ -0,0 +1,26 @@
+From e7f35d4a775f2ba372c2b7e90bbe9baef8e25d8e Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Fri, 20 Sep 2019 16:02:57 +0000
+Subject: [PATCH] Pin newtonsoft json to a specific version
+
+---
+ build/config.props | 3 +--
+ 1 file changed, 1 insertion(+), 2 deletions(-)
+
+diff --git a/build/config.props b/build/config.props
+index 1332abb..52176e0 100644
+--- a/build/config.props
++++ b/build/config.props
+@@ -48,8 +48,7 @@
+ 
+   <!-- Dependency versions -->
+   <PropertyGroup>
+-    <NewtonsoftJsonVersionCore Condition="$(NewtonsoftJsonPackageVersion) == ''">9.0.1</NewtonsoftJsonVersionCore>
+-    <NewtonsoftJsonVersionCore Condition="$(NewtonsoftJsonPackageVersion) != ''">$(NewtonsoftJsonPackageVersion)</NewtonsoftJsonVersionCore>
++    <NewtonsoftJsonVersionCore>9.0.1</NewtonsoftJsonVersionCore>
```

Comment:
Can this version be defined in common props file (for source-built packages)?

Reply:
This should be a stable version, not floating, because we're building a newer version (12.0.2), which was getting picked up, but they're using functionality from the 9.0.1 version